src/token-helper.js

```diff
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 const { Ims, ACCESS_TOKEN, REFRESH_TOKEN } = require('./ims')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-lib-ims:token-helper', { provider: 'debug' })
 const { getContext } = require('./context')
-const imsJwtPlugin = require('@adobe/aio-lib-ims-jwt')
+const imsJwtPlugin = '@adobe/aio-lib-ims-jwt'
```

Comment:
Have you had the chance to test the ims lib in a runtime action? Otherwise we will test it, the reason we do a static require is to let webpack optimizer know it has to include the `'@adobe/aio-lib-ims-jwt'` module when bundling/compiling the action code

Reply:
Nope, only tested the CLI, actually.

That is a good point regarding web pack. Can web pack be instructed to consider this dependency also in another way ?

If not we could revert this to be a static import and adapt the [`loadPlugin`](https://github.com/adobe/aio-lib-ims/pull/76/files#diff-64d7a2f023f1c25cefceb40c0dd6b2098bdf46c03ae7f7979cd7d31bbced1f2dR79) function accordingly.

Reply:
I'm trying to understand better how web pack is working, but a quick and dirty solution to this issue would be :

```suggestion
diff --git a/src/token-helper.js b/src/token-helper.js
index c2c37ae..9b774a3 100644
--- a/src/token-helper.js
+++ b/src/token-helper.js
@@ -13,7 +13,7 @@ governing permissions and limitations under the License.
 const { Ims, ACCESS_TOKEN, REFRESH_TOKEN } = require('./ims')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-lib-ims:token-helper', { provider: 'debug' })
 const { getContext } = require('./context')
-const imsJwtPlugin = '@adobe/aio-lib-ims-jwt'
+const imsJwtPlugin = require('@adobe/aio-lib-ims-jwt')
 const { codes: errors } = require('./errors')
 
 /**
@@ -80,6 +80,10 @@ async function getMergedPlugins (context) {
 function loadPlugin (name, location) {
   aioLogger.debug('loadPlugin(%s, %s)', name, location)
 
+  if (name === 'jwt') {
+    return imsJwtPlugin
+  }
+
   try {
     return require(location)
   } catch (error) {
```